src/pages/about.html

```diff
@@ -0,0 +1,222 @@
+            </a>
+            <a href="#" class="nav-popup-list-link">
+              <li class="nav-popup-list-item">> News</li>
+            </a>
+          </ul>
+        </section>
+        <section class="nav-desktop desktop-only">
+          <ul class="nav-links-list">
+            <div class="nav-links-list-logo">
+              <a href="../../index.html" class="nav-links-list-logo-link">
+                <li class="nav-links-list-item">
+                  <img
+                    src="../assets/images/codingConLogo.png"
+                    alt="codingCon logo"
+                    class="nav-links-list-logo"
+                  />
+                </li>
+              </a>
+            </div>
+            <div class="nav-links-list">
+              <a href="#" class="nav-links-list-link">
+                <li class="nav-links-list-item active-page">About</li>
+              </a>
+              <a href="#" class="nav-links-list-link">
+                <li class="nav-links-list-item">Program</li>
+              </a>
+              <a href="#" class="nav-links-list-link">
+                <li class="nav-links-list-item">Join</li>
+              </a>
+              <a href="#" class="nav-links-list-link">
+                <li class="nav-links-list-item">Sponsor</li>
+              </a>
+              <a href="#" class="nav-links-list-link">
+                <li class="nav-links-list-item">News</li>
+              </a>
+            </div>
+          </ul>
+        </section>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="about-main-intro main-intro">
+        <p class="main-intro-preheading about-intro-preheading">
+          "Hello World!"
+        </p>
+        <h1 class="main-intro-heading about-intro-heading">CodingCon 2023</h1>
+        <p class="about-intro-description">
+          CodingCon brings together the world's leading experts, developers,
+          designers, and business leaders who are shaping the future of
+          technology. Join us for three days of inspiration, learning, and
+          connection. We can't wait to see you in 2023!
+          <br /><br /><br /><br />
+          Please contact us via email @<a href="mailto:codingcon@codingcon.org"
+            >codingCon@codingCon.org</a
+          >
+          for any further questions.
+        </p>
+      </section>
+
+      <section class="about-codingcon-logo desktop-only">
+        <h2 class="about-codingcon-logo-heading">CodingCon logo 2023</h2>
+        <hr class="about-codingcon-logo-hr hr-orange" />
+        <p class="about-codingcon-logo-description">
+          The logo of CodingCon 2023 was decided through a competition among
+          designers from all over the world. The logo was decided based on the
+          logo competition's results.
+        </p>
+        <img
+          class="about-codingcon-logo-img"
+          src="../assets/images/codingConLogo.png"
+          alt="codingCon logo"
+        />
+      </section>
```

Comment:
- I think it will be better if we made the background color of this section a LITTLE BIT different from the header. This is just for user experience and to follow behance color design guidelines. 🤗 
- The `about-codingcon-logo desktop-only` isn't reflected on your mobile version. Can we fix that??

This is what i could see 👇🏽 

| Your Mobile Logo  |Behance Logo section |
| :---:   | :---: | 
| <img width="382" alt="Screenshot 2023-01-04 at 10 43 07" src="https://user-images.githubusercontent.com/99246359/210595045-723848be-7a57-4f2b-a7a8-e2bf1b2a1797.png"> |!<img width="579" alt="Screenshot 2023-01-04 at 10 44 55" src="https://user-images.githubusercontent.com/99246359/210595098-9beb3653-c8be-40e9-b4dd-b946f5e90ffe.png">  |